src/views/tutorial-view/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function splitProductFromFilename(slug: string): string {
+  return slug.split('/')[1]
```

Comment:
This is an example of some of the slug helpers we'll need to have to handle the discrepancies with the db slugs.